client-app/ui-kit/components/molecules/quantity/vc-quantity.vue

```diff
@@ -83,6 +83,10 @@ async function onQuantityChanged(): Promise<void> {
     return;
   }
 
+  if (quantity.value > LINE_ITEM_QUANTITY_LIMIT) {
+    quantity.value = Number(quantity.value.toString().slice(0, 6));
```

Comment:
I think it's better to use String(LINE_ITEM_QUANTITY_LIMIT).length instead of hardcoded value 6